book/02-using-atom/sections/02-text.asc

```diff
@@ -338,7 +338,7 @@ image::images/autocomplete.png[autocomplete]
 
 By default, the autocompleter will look through the current open file for strings that match what you're starting to type.
 
-If you want more options, in the Settings panel for the Autocomplete package you can toggle a setting to make the autocompleter look for strings in all your open buffers rather than just the current file.
+If you want more options, in the Settings panel for the Autocomplete-plus package you can toggle a setting to make the autocompleter look for strings in all your open buffers rather than just the current file.
```

Comment:
I think we should go with autocomplete-plus - the humanized name is only used for the breadcrumb in settings-view now and I was thinking about removing it.